docs/usage_guide.rst

```diff
@@ -387,7 +388,27 @@ Trade settings
 Crypto to crypto trades
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Specify whether crypto to crypto trades are taxable and should be taken into account. If yes then each crypto to crypto trade also creates a "virtual" trade that sells or buys the crypto asset for fiat and then sells or buys the fiat for the other crypto asset.
+A setting to determine whether crypto to crypto trades are taxable and should be taken into account. By default it's ``True``.
+
+Illustration: You have ``1`` ETH that you bought with price ``50 EUR``. Then you spend this ``1`` ETH for gas fees, and the price at the moment is ``100 EUR``.
+
+
+- ``True``: When you spending crypto (as in fees), a sell is also generated, and this sell can generate additional :green:`profit` or :red:`loss`. Each crypto to crypto trade creates a "virtual" trade that sells or buys the crypto asset for fiat and then sells or buys the fiat for the other crypto asset.
+
+..
+
+    Profit of this sell is ``100 - 50`` = ``50 EUR``.
+
+    Because you spend this ETH, you also sell this ETH for ``100 EUR``, which means you lost ``100 EUR``.
+    So total PnL of this action is ``50 - 100`` EUR = ``-50 EUR``
+
+
+- ``False``: The above does not happen.
+
+..
+
+    So total PnL of this action will be calculated as ``100 - 50`` = ``50 EUR``.
```

Comment:
I think I make mistake in this part. It should be just `-100EUR` instead. Will fix it